lldb/source/Plugins/Process/Linux/NativeRegisterContextLinux_arm64.h

```diff
@@ -234,9 +234,9 @@ class NativeRegisterContextLinux_arm64
 
   size_t GetFPMRBufferSize() { return sizeof(m_fpmr_reg); }
 
-  llvm::Error ReadHardwareDebugInfo() override;
+  Status ReadHardwareDebugInfo() override;
```

Comment:
@labath I seem to remember you saying we want to move to or away from Status, if I didn't imagine that, which is it?

Reply:
Because I noticed that the function calling it returns a `Status` (`virtual Status ClearAllHardwareWatchpoints()` in file lldb/include/lldb/Host/common/NativeRegisterContext.h), and other architectures, such as ARM (not AArch64) and PPC, also return a Status.
Should consistency be maintained here? If `llvm::Error` is required, I will revert it. I also made a mistake—the definition in this file `NativeRegisterContextFreeBSD_arm64.h` hasn't been updated yet. I apologize for my carelessness.